src/features/comments/commentsSlice.ts

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable @typescript-eslint/indent */
+/* eslint-disable no-param-reassign */
+import { createSlice, createAsyncThunk, PayloadAction } from '@reduxjs/toolkit';
+import { Comment, CommentData } from '../../types/Comment';
```

Comment:
Ensure that the `Comment` and `CommentData` types are correctly defined in the `types/Comment` file.